.pre-commit-config.yaml

```diff
@@ -0,0 +1,33 @@
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v2.4.0
+    hooks:
+      - id: trailing-whitespace
+      - id: end-of-file-fixer
+      - id: check-yaml
+      - id: check-added-large-files
+  - repo: https://github.com/asottile/seed-isort-config
+    rev: v1.2.0
```

Comment:
Should we consider updating these pre-commit hooks? For example lates seed-isort-config is v2.2.0 and latest pre-commit is v3.2.0.

Reply:
I don't see any reason why not to do it. Updated